docs/getting-started.md

```diff
@@ -0,0 +1,168 @@
+
+Similarly, for regions, `client.lookup('regions', 1215)[contains]` will return a list of region ids for regions that are in the US (region_id = 1215): `[13100, 13061, 13053, 13099, ....]`. And each of those can be further looked up e.g. `client.lookup('regions', 13100)` will return `{'name': 'Wisconsin', 'level': 4, 'contains': [139839, 139857, 139863, ...]}`.
+
+### Get descendants
+The Lookup Contains method can return a list of an entity's direct children. However, you may wish to discover all of the lower level regions that belong to a high level region. For this special case of traversing regions, there's a shortcut function that does it more directly and gives the option of filtering by region level: `get_descendant_regions(region_id, descendant_level)`.
+
+Where:
+```
+provinces_of_brazil = client.get_descendant_regions(1029, 4)
+```
+will recursively look up all descendants of region 1029 (Brazil) that are of level 4 (provinces) i.e., all the provinces of Brazil. 
+
+And: 
+```
+districts_of_brazil = client.get_descendant_regions(1029, 5)
+```
+will recursively look up all descendants of region 1029 (Brazil) that are of level 5 (districts), i.e., all the districts in all the provinces of Brazil.
+
+For more information on region levels, please refer to the [Special properties for regions](../docs/gro-data-key-concepts#special-properties-for-regions) section of [Gro Data: Key Concepts]((../docs/gro-data-key-concepts)
+
+### Lookup belongs
+If you want to find "what entities contain the given entity?" there is a helper function, `client.lookup_belongs(type, child_id)` to does just that. For example:
+
+`client.lookup_belongs('regions', 1215)`
+will return `[{id: 15, name: 'North America', contains: [1215, 1037, ...], level: 2}, ...]`
+
+## Retrieving Data
+Once you've identified the data you want, you'll want to start retrieving it and putting it to use. The following methods should help you get the data in the format that you want.
+
+### Get data points
+`get_data_points(**selection)` is the primary method for retrieving data. The [code snippets](#code-snippets) feature covered earlier provides you with a fully completed `get_data_points` query, such as:
+```# Wheat - Area Harvested (area) - India (USDA PS&D)
+client.get_data_points(**{'metric_id': 570001, 'item_id': 95,'region_id': 1094, 'source_id': 14, 'frequency_id': 9})
+```
+The above query has completed fields for `metric_id`, `item_id`, `region_id`, `source_id`, and `frequency_id`. However, `get_data_points` can also accept fields to further narrow your data series of interest: `partner_region_id` (used only in series that represent a flow between two places), `start_date`, `end_date`, and [`show_revisions`](#show-revisions).
+
+At a minimum, get_data_points must have a `metric_id` and a `source_id` specified. Omitting some of the default fields can help you retrieve greater amounts of data in a single query. For example, removing the `item_id` from the above query:

```

Comment:
Let's get rid of this paragraph. This is not guaranteed behavior and will probably be removed. Should be using `get_data_series()` with partial selections and only passing complete selections into `get_data_points()`

Reply:
Good call